cfme/middleware/deployments.py

```diff
@@ -0,0 +1,13 @@
+from cfme.fixtures import pytest_selenium as sel
+from cfme.middleware.provider import HawkularProvider
+
+
+class Deployments(HawkularProvider):
+
+    def __init__(self, name):
+        self.name = name
+        self.provider = self.string_name
+
+    def nav_to_deployments_view(self):
```

Comment:
Deployments 

----- Original Message -----

> From: "Milan Falešník" <notifications@github.com>
> To: "RedHatQE/cfme_tests" <cfme_tests@noreply.github.com>
> Cc: "mattmahoneyrh" <mmahoney@redhat.com>, "Author"
> <author@noreply.github.com>
> Sent: Thursday, April 21, 2016 9:55:47 AM
> Subject: Re: [RedHatQE/cfme_tests] MW Provider View Classes - For Navigation
> and Validation Methods (#2877)

> In cfme/middleware/deployments.py :
> > @@ -0,0 +1,13 @@
> > +from cfme.fixtures import pytest_selenium as sel
> > +from cfme.middleware.provider import HawkularProvider
> > +
> > +
> > +class Deployments(HawkularProvider):
> > +
> > +    def __init__(self, name):
> > +        self.name = name
> > +        self.provider = self.string_name
> > +
> > +    def nav_to_deployments_view(self):

> I haven't seen the UI for this but is the name of the thing Deployments or
> just Deployment?

> Anyway, the nav_to_deployments_view should be rather called load_details to
> align with the interface of other objects.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub